setup.py

```diff
@@ -4,9 +4,13 @@
 
 setup(
     use_scm_version=True,
-    name='sstable_tools',
-    version='1.0.0',
-    description='Scylla SStable Tools',
-    packages=['sstable_tools'],
-    scripts=['sstable-compressioninfo.py', 'sstable-index.py', 'sstable-statistics.py', 'sstable-summary.py'],
+    name="sstable_tools",
```

Comment:
Are these changes done by some automatic formatter?

Reply:
yes,
I was using black
https://github.com/psf/black

it was recently adopted by the Python Foundation